common/plugins/org.jboss.tools.common.jdt.debug/src/org/jboss/tools/common/jdt/debug/tools/ToolsCore.java

```diff
@@ -95,7 +96,17 @@ public Object getMonitoredVM() {
 	}
 	
 	public static boolean processIsRunning(String hostname, int vmPid) throws ToolsCoreException {
-		return Tools.getInstance().getMonitoredVm(hostname, vmPid) != null;
+		// If these throw exceptions, we want them to be rethrown
+		Object host = Tools.getInstance().invokeGetMonitoredHost(hostname);
+		Object vmId = Tools.getInstance().invokeVmIdentifier(String.format(IToolsConstants.VM_IDENTIFIRER, vmPid));
+		
+		// We want to catch and ignore these failures and just return that the process is not running
+		try {
+			Object monitoredVm = Tools.getInstance().invokeGetMonitoredVm(host, vmId);
+			return monitoredVm != null;
+		} catch(ToolsCoreException tce) {
```

Comment:
This code sure would fix problem with extra logging on console but side effects are unpredictable, in most cases we would not be able to do support, because piece of useful information would be missing. For example invokeGetMonitoredVm(host,vmId) call could trigger 6 types of exceptions:
``` java
		} catch (InvocationTargetException t) {
		} catch (IllegalAccessException e) {
		} catch (IllegalArgumentException e) {
		} catch (NoSuchMethodException e) {
		} catch (SecurityException e) {
		} catch (ClassNotFoundException e) {
```
Only one InvocationTargeException related to linked issue other are different scenarios. For example it would probably throw ClassNotFoundException for trying to monitor java < 1.5 (not sure if they appear in the list though). So instead of showing the warning about supporting only Java 1.5+ we just say: "OMG! Process is not running", which would be false statement.